examples/pct2075_test/pct2075_test.ino

```diff
@@ -11,7 +16,7 @@ void setup() {
   while (!Serial) { delay(1); }
   Serial.println("Adafruit PCT2075 Test");
 
-  if (!PCT2075.begin()) {
+  if (!PCT2075.begin(i2c)) {
```

Comment:
@TheKitty  Replace `i2c` with `PCT2075_I2CADDR_DEFAULT` here.

Reply:
You're right, I was distracted by taking Bowie out to do his business with ICE in the area